Fixed code indent and padding

tuantinghuang opened the pull request:
Relate #349 #345 

<img width="949" alt="Screenshot 2023-08-02 at 3 26 39 PM" src="https://github.com/nature-of-code/noc-book-2023/assets/90000947/a91d923d-4679-40c6-a797-2abad8c6ded0">

tuantinghuang committed:
Fix code indent and padding

shiffman committed:
Merge branch 'main' into fix-code-indent-and-padding

shiffman commented:
This looks great! One small issue now, however, is that the vertical padding is such that when there is code text on two lines one above the other, they blend together. Maybe we should reduce this by a pixel or two?

![image](https://github.com/nature-of-code/noc-book-2023/assets/191758/61eca2ec-7fd1-47d3-b4f9-2c700f7988a2)

Also, what do you think about extending the grey code block highlighting by a little bit?

![image](https://github.com/nature-of-code/noc-book-2023/assets/191758/9085ec10-21aa-4a9a-a9c3-7d50f6078e80)

shiffman committed:
Merge branch 'main' into fix-code-indent-and-padding

shiffman committed:
Merge branch 'main' into fix-code-indent-and-padding

tuantinghuang committed:
Merge branch 'main' into fix-code-indent-and-padding

tuantinghuang committed:
update coding styling

Fixed the grey code blocks' left and right padding + the inline code's vertical padding.

tuantinghuang commented on magicbook/stylesheets/components/typography.scss:
Adding some padding to the grey code blocks and fixed the inline code's vertical padding.

<img width="721" alt="Screenshot 2023-08-09 at 8 29 06 PM" src="https://github.com/nature-of-code/noc-book-2023/assets/90000947/621bbee1-9379-4abf-95d7-254cfa14aa6b">

shiffman replied:
This is looking great!

shiffman commented:
Merging! We can close #345 and #349 but maybe we want to wait and check how it looks after finalize #348.
